src/Views/Game.hs

```diff
@@ -120,10 +120,13 @@ addGame api gameID = do
       appendChild gameListDiv gameDiv
 
       clickEventString gameName $ do
```

Comment:
`clickEventString` should really change name something more like `onElemByStringMouseClick` covers it pretty well but it feels to verbose. Maybe just `onElemMouseClick`. But that change isn't for this pull request or branch.